timetracker/timetracker/urls.py

```diff
@@ -15,11 +15,24 @@
 """
 from django.conf.urls import include, url
 from django.contrib import admin
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpRequest
 
 
 def hello_world(request):
-    return HttpResponse('Hello World')
+	visitorName = request.GET.get("name", False)
+	responseString = ""
+	if visitorName == False:
```

Comment:
A caveat to that, if the user enters an empty value, it, too, is considered falsey... which may be what you want [as in this case], but is worth remembering.